src/routes/namespaces/[namespace]/workflows/_workflow-filters.svelte

```diff
@@ -27,9 +29,11 @@
   };
 </script>
 
-<section class="p-4 flex flex-col">
+<section class="flex flex-col my-8">
   <h2 class="text-3xl mb-4">Workflows</h2>
-  <div class="flex gap-4">
+  <div class="grid grid-cols-5 gap-4">
+    <FilterInput id="workflow-id-filter" name="Workflow ID" value="" />
```

Comment:
why do we need these ids? didn't find anywhere in code

Reply:
Mostly for accessibility.